### PR TITLE
Corrected "peform", close #775

### LINKED ISSUE #775: Spelling error "peform"
⚠️ Open Source Workshop ⚠️

Should be "perform"

https://github.com/Instagram/IGListKit/blob/master/Guides/Best%20Practices%20and%20FAQ.md

Also a "perfom" in
https://github.com/Instagram/IGListKit/blob/master/CHANGELOG.md

@rnystrom

### PR DESCRIPTION
**Pull request by heshamsalman**
## Changes in this pull request

Issue fixed: #775

### Checklist

- [ ] All tests pass. Demo project builds and runs.
- [ ] I added tests, an experiment, or detailed why my change isn't tested.
- [ ] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [ ] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md)
#trivial

### CODE REVIEW

**Comment by Sherlouk** [comment]
@heshamsalman Hope you don't mind piggy backing off of your PR with the other spelling mistake listed in 775

**Comment by heshamsalman** [comment]
@Sherlouk no worries 😁

File: docs/best-practices-and-faq.html [hunk 1]
```diff
@@ -300,9 +300,9 @@ <h4 id='what-39-s-the-purpose-of-code-iglistcollectionview-code' class='heading'
...
+<p>If you have multiple thousands of items and you cannot batch them in, you&rsquo;ll see performance issues with <code>-performUpdatesAnimated: completion:</code>. The real bottle neck behind the scenes here is <code>UICollectionView</code> attempting to insert so many cells at once. Instead, call <a href="https://instagram.github.io/IGListKit/Classes/IGListAdapter.html#/c:objc(cs)IGListAdapter(im)reloadDataWithCompletion:"><code>-reloadDataWithCompletion:</code></a> when you first load data, which will be super fast. Behind the scenes, this method <em>does not</em> do any diffing and simply calls <code>-reloadData</code> on <code>UICollectionView</code>. For subsequent updates, you can then use <code>-performUpdatesAnimated: completion:</code>.</p>
```
Comment (jessesquires) [inline docs/best-practices-and-faq.html]: we'll need to revert this and make the changes in `Guides/Best Practices and FAQ.md` instead 😄

**Comment by heshamsalman** [comment]
Ha, side effects of running `sed` indiscriminately 😂

**Pull request closed without merging**